src/Spago/Build.hs

```diff
@@ -323,11 +323,14 @@ runBackend
 runBackend maybeBackend RunDirectories{ sourceDir, executeDir } moduleName maybeSuccessMessage failureMessage extraArgs = do
   logDebug $ display $ "Running with backend: " <> fromMaybe "nodejs" maybeBackend
   BuildOptions{ pursArgs } <- view (the @BuildOptions)
-  isES <- Purs.hasMinPursVersion "0.15.0"
+  isES <- Purs.hasMinPursVersion "0.15.0-alpha-01"
   let postBuild = maybe (nodeAction isES $ Path.getOutputPath pursArgs) backendAction maybeBackend
   build (Just postBuild)
   where
     fromFilePath = Text.pack . Turtle.encodeString
+    runJsSource isES =
+      let file = if isES then "run.mjs" else "run.js"
```

Comment:
Could this be updated to use `.js` for both CJS and ES modules as we did in `pulp`? It requires adding a `package.json` file to the same dir that stores the `run.js` file.

On Node.js <`13.0.0`, we'd also need to add the `--experimental-modules` flag before calling the file in the script. For example, `node --experimental-modules run.js`.

Reply:
Is there any advantage in adding a `package.json` the `.spago` folder?

Good catch, I’m not sure though we have the node version here 🤔

Reply:
> Is there any advantage in adding a package.json to the .spago folder over using the extensions?

:man_shrugging: globs will always use a single extension?

Reply:
Fixed